README.md

```diff
@@ -1,2 +1,168 @@
-# School-Library-DB
-A school library project using ruby.
+<a name="readme-top"></a>
+
+  <h1><b>School-Library</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 <a name="about-project"></a>
+
+> School library is...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech used in thsi project...
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="/">Ruby</a></li>
+  </ul>
+</details>
+### Key Features <a name="key-features"></a>
+
+- School library app...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] It would be nice if you remove the Live Demo section since you don't have any working live link right now 🤔 Whenever you would have a working live link then you can add this section again in your readme to make your readme descriptive and professional 👍

Reply:
### @HAMMAS-SALEEM  Hey there,👋 thank you so very much for making my PR `invalid` 🔴 for this great major issue. Maybe you have missed to see the text `- N/A` written next to it, well, for your satisfaction I will gladly remove that section. Please, I hope you will review my PR again.